docs/Developers/Git_Basics.rst

```diff
@@ -329,7 +329,7 @@ to understand how it works.
 Rebasing
 --------
 
-As people work on coala new commits will be added. This will result in your
+As people work on coala, new commits will be added. This will result in your
 local fork going out of sync with the remote repository.
 To sync your changes with the remote repository run the following commands in
 the desired branch:
```

Comment:
Actually, you're editing the very doc that you need to use 😄 
Maybe follow this very doc and see if you can come up with something?
`git rebase -i [...]` is your friend, let me know if you need help!